src/actions/contactsActions.js

```diff
@@ -48,22 +48,26 @@ import type { Dispatch, GetState } from 'reducers/rootReducer';
 
 export const searchContactsAction = (query: string) => {
   return async (dispatch: Dispatch, getState: GetState, api: Object) => {
-    const { user: { data: { walletId } }, contacts: { data: localContacts } } = getState();
+    const {
+      user: { data: { walletId } },
+      contacts: { data: localContacts },
+    } = getState();
+    const upperCaseQuery = query.toUpperCase();
+
+    const myContacts = localContacts.filter(({ username }) => {
```

Comment:
```suggestion
    const myContacts = localContacts.filter(({ username }) => username.toUpperCase().includes(upperCaseQuery));
```

Reply:
Or you can even use `isCaseInsensitiveMatch` from utils.

Reply:
Well actually, no, `isCaseInsensitiveMatch` doesn't provide the same behaviour :grimacing: